On OSX, math functions aren't found by the default toolchain

Building the viewer yields:

vibesgraphicsitem.cpp:1288:34: error: no member named 'cos' in namespace
      'std'; did you mean simply 'cos'?
        double m1_x = cx + rho_m*std::cos(-theta_m*M_PI/180.0);

Seems to be due to a missing #include <cmath>

--- a/viewer/vibesgraphicsitem.cpp
+++ b/viewer/vibesgraphicsitem.cpp
@@ -14,6 +14,8 @@
 #include <QJsonArray>
 #include <QJsonValue>
 
+#include <cmath>
+
 // The only instance of VibesDefaults
 VibesDefaults VibesDefaults::_instance;
 

